app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -7,6 +7,11 @@ def show
       @application_form = current_application
     end
 
+    def apply_2
+      DuplicateApplication.new(current_application).duplicate
+      redirect_to candidate_interface_application_form_path
+    end
```

Comment:
This probably needs some checking to prevent double submission / back button from generating 2+ new applications. For example, we could guard against duplicating if there already is an unsubmitted application.